[created accidentally please ignore]

calebfoss opened the pull request.

limzykenneth commented:
The magnifying glass emoji is too out of place and attention grabbing. Instead of making it part of the placeholder, can it instead be an `:after` element attached to the search bar with a simple image background and aligned to the right side of the search bar? 

It doesn't need to be hidden even when the user is typing on the box, just need to make sure the text doesn't overlap with the image.

pcgamer1 commented:
@limzykenneth I am working on it.

pcgamer1 commented:
@limzykenneth How does this look?
![newsearch](https://user-images.githubusercontent.com/30899040/79346865-8002cd80-7f50-11ea-80f6-4032f202e404.PNG)

limzykenneth commented:
I'd prefer the magnifying glass to be a simple monochrome image and for it to be inside the box.

pcgamer1 commented:
@limzykenneth working on that.

pcgamer1 commented:
@limzykenneth How does this look?
![newsearch](https://user-images.githubusercontent.com/30899040/79350797-83e51e80-7f55-11ea-8c93-17c1a67b227b.PNG)

limzykenneth commented:
Much better, maybe make the image's outline color match the placeholder text's color.

pcgamer1 commented:
@limzykenneth I haven't loaded an image in for the icon, it's just an unicode character rotated towards right. I don't think I'll be able to change the outline. Should I try a loading an image if you want the outline to be grey?

yunyoungJang committed:
Create bezierDetail.mdx

yunyoungJang committed:
Update bezierDetail.mdx

KO translation

yunyoungJang committed:
Delete bezierDetail.mdx

Tmicrd committed:
Foundation

Harrycheng233 committed:
Update lights.mdx

Tmicrd committed:
Updated

Harrycheng233 committed:
Update orbitControl.mdx

Harrycheng233 committed:
Update debugMode.mdx

Harrycheng233 committed:
Update noDebugMode.mdx

Harrycheng233 committed:
Update ambientLight.mdx

Harrycheng233 committed:
Update specularColor.mdx

Harrycheng233 committed:
Update directionalLight.mdx

Harrycheng233 committed:
Update pointLight.mdx

Harrycheng233 committed:
Update imageLight.mdx

Harrycheng233 committed:
Update panorama.mdx

Harrycheng233 committed:
Update lightFalloff.mdx

Harrycheng233 committed:
Update spotLight.mdx

Harrycheng233 committed:
Update noLights.mdx

Harrycheng233 committed:
Update loadShader.mdx

Harrycheng233 committed:
Update createShader.mdx

Harrycheng233 committed:
Update createFilterShader.mdx

Harrycheng233 committed:
Update shader.mdx

Harrycheng233 committed:
Update resetShader.mdx

yunyoungJang committed:
Create curvePoint.mdx

yunyoungJang committed:
Delete src/content/reference/ko/p5/src/content/reference/en/p5/curvePoint.mdx

yunyoungJang committed:
Create beginShape.mdx

yunyoungJang committed:
Delete src/content/reference/ko/p5/src/content/reference/en/p5/beginShape.mdx

yunyoungJang committed:
Create bezierVertex.mdx

yunyoungJang committed:
Delete src/content/reference/ko/p5/src/content/reference/en/p5/bezierVertex.mdx

ienground committed:
Update camera.mdx

ienground committed:
Update camera.mdx

Harrycheng233 committed:
Update TEXTURE.mdx

Harrycheng233 committed:
Update textureMode.mdx

Harrycheng233 committed:
Update textureWrap.mdx

Harrycheng233 committed:
Update normalMaterial.mdx

Harrycheng233 committed:
Update ambientMaterial.mdx

Harrycheng233 committed:
Update emissiveMaterial.mdx

Harrycheng233 committed:
Update specularMaterial.mdx

Harrycheng233 committed:
Update shininess.mdx

Harrycheng233 committed:
Update metalness.mdx

douMax committed:
translate 3d

douMax committed:
Update camera.mdx

douMax committed:
Update ortho.mdx

douMax committed:
Update frustum.mdx

douMax committed:
Update camera.mdx

douMax committed:
Update centerX.mdx

douMax committed:
Update centerX.mdx

douMax committed:
Update centerY.mdx

douMax committed:
Update centerZ.mdx

douMax committed:
Update eyeX.mdx

douMax committed:
Update eyeY.mdx

douMax committed:
Update eyeZ.mdx

douMax committed:
Update lookAt.mdx

douMax committed:
Update move.mdx

douMax committed:
Update pan.mdx

douMax committed:
Update perspective.mdx

douMax committed:
Update set.mdx

douMax committed:
Update setPosition.mdx

douMax committed:
Update slerp.mdx

douMax committed:
Update tilt.mdx

douMax committed:
Update upX.mdx

douMax committed:
Update upY.mdx

douMax committed:
Update upZ.mdx

EshaanAgg committed:
spell check and change default comment

yunyoungJang committed:
Update endGeometry.mdx

yunyoungJang committed:
Update loadModel.mdx

Harrycheng233 committed:
Update TEXTURE.mdx

Harrycheng233 committed:
Update metalness.mdx

Harrycheng233 committed:
Update TEXTURE.mdx

Tmicrd committed:
Updated

Harrycheng233 committed:
Update shininess.mdx

Harrycheng233 committed:
Update specularMaterial.mdx

Harrycheng233 committed:
Update emissiveMaterial.mdx

Harrycheng233 committed:
Update ambientMaterial.mdx

Harrycheng233 committed:
Update normalMaterial.mdx

Harrycheng233 committed:
Update textureWrap.mdx

Harrycheng233 committed:
Update textureMode.mdx

Harrycheng233 committed:
Update resetShader.mdx

Harrycheng233 committed:
Update shader.mdx

Harrycheng233 committed:
Update createFilterShader.mdx

Harrycheng233 committed:
Update createShader.mdx

Harrycheng233 committed:
Update loadShader.mdx

Harrycheng233 committed:
Update noLights.mdx

Harrycheng233 committed:
Update spotLight.mdx

Harrycheng233 committed:
Update lightFalloff.mdx

Harrycheng233 committed:
Update panorama.mdx

Harrycheng233 committed:
Update imageLight.mdx

Tmicrd committed:
Foundation: translated module and submodule

Harrycheng233 committed:
Update pointLight.mdx

Harrycheng233 committed:
Update directionalLight.mdx

Harrycheng233 committed:
Update specularColor.mdx

Harrycheng233 committed:
Update ambientLight.mdx

Harrycheng233 committed:
Update noDebugMode.mdx

Harrycheng233 committed:
Update debugMode.mdx

Harrycheng233 committed:
Update orbitControl.mdx

Harrycheng233 committed:
Update lights.mdx

Harrycheng233 committed:
Update debugMode.mdx

Tmicrd committed:
Fixed the format of string.mdx

Tmicrd committed:
Fixed the format of let.mdx

Tmicrd committed:
Fixed the format of for.mdx

Tmicrd committed:
Fixed the format of if.mdx

Tmicrd committed:
Fixed the format of number.mdx

Tmicrd committed:
Fixed the format of bezier.mdx

ranaaditya committed:
add: hi translation

yinhwa committed:
Update endGeometry.mdx

yinhwa committed:
Update loadModel.mdx

designerSejinOH committed:
Update specularColor.mdx

designerSejinOH committed:
Update directionalLight.mdx

designerSejinOH committed:
Update panorama.mdx

designerSejinOH committed:
Update lights.mdx

designerSejinOH committed:
Update spotLight.mdx

designerSejinOH committed:
Update let.mdx

Tmicrd committed:
updated

jaiakash committed:
revert: 3D as english

jaiakash committed:
fix: coordinate to hindi

jaiakash committed:
fix: material to hindi

douMax committed:
traslated type

jaiakash committed:
revert changes to files

yinhwa committed:
Update TEXTURE.mdx

outofambit committed:
slow asterisk spin speed

outofambit committed:
set top nav's top gap to 20px

davepagurek committed:
Comment out OP API things because im on a plane

davepagurek committed:
Switch to using a light code theme

davepagurek committed:
Let multi-column content go larger than the column width

davepagurek committed:
Use ui translation for callout titles

davepagurek committed:
More style fixes for custom components in tutorials

davepagurek committed:
Make callouts readable in dark theme

davepagurek committed:
Clean up code styles in dark theme

davepagurek committed:
Revert "Comment out OP API things because im on a plane"

This reverts commit 509cb364c39ee25fc84fcc2564cd9e0d72e51970.

davepagurek committed:
Fix extra spaces appearing in code blocks

outofambit committed:
set heights for root page headers to match sidebar

outofambit committed:
homepage header height

outofambit committed:
fix

outofambit committed:
add x button to filter ui

outofambit committed:
write people import script

outofambit committed:
add contributors entries

outofambit committed:
Revert "add contributors entries"

This reverts commit 89bfdc133561e7df877c2343e576c88b2cef75c2.

outofambit committed:
re-add contributors

outofambit committed:
nav panels exclusive responsively

outofambit committed:
keep mobile nav sticky on scroll (#347)

outofambit committed:
rename

davepagurek committed:
Rename non-constant reference files to lowercase

davepagurek committed:
Put back rich descriptions in class methods/properties

outofambit committed:
larger text for about page

outofambit committed:
mobile menu text styles

outofambit committed:
remove excess space at bottom of rendered markdown areas

outofambit committed:
fix bug from #351

outofambit committed:
re-do fix

d216e9338af4c443c89137ec073ed52ac307be59

outofambit committed:
nav menu arrow padding

outofambit committed:
full bleed divider on about page

outofambit committed:
jump to menu item spacing

outofambit committed:
Update styles.module.scss

outofambit committed:
type cleanup

outofambit committed:
fix people script bug for names with slashes

outofambit committed:
First pass at docs (#355)

* add basic technical overview

* reorder npm tasks

* Create localization.md

* Create styles_architecture.md

* Create scripts.md

* remove dead npm scripts

* update script names

* create docs folder

davepagurek committed:
Add fixes for sketch thumbnails and layouts

davepagurek committed:
Fix type errors

davepagurek committed:
Cast to any because apaprently lodash types aren't working

davepagurek committed:
Move sketch images to the api folder

davepagurek committed:
Add banner

outofambit committed:
add script to pull in latest p5 version

and use across website pages

outofambit committed:
add 2 placeholder images for homepage hero

davepagurek committed:
Make banner its own collection

davepagurek committed:
Fix type errors

davepagurek committed:
Display images with alts on event pages

davepagurek committed:
Fix type error

Qianqianye committed:
Add home page hero images

Qianqianye committed:
Fix image link

davepagurek committed:
Fix more type errors

davepagurek committed:
Put back expect-error lines

Qianqianye committed:
Update images on donate page

calebfoss committed:
Remove Instance_Container missing history

calebfoss committed:
Add ignore suffix to Instance_Container

calebfoss committed:
Revert "Restore example history"

Commit:
Added spanish setup readme

Added a Spanish translated version of the setup() function. I also made sure that "Structure" module andf submodule is referred to as "Structure" In other Spanish translations on the website

limzykenneth committed:
Update reference to latest version

limzykenneth committed:
Fix reference on types clashing with type conversion functions

lauradowell committed:
Updated draw.mdx

used juan's clarity suggestions

lauradowell committed:
Update draw.mdx

fixed yaml format issue

lauradowell committed:
Update draw.mdx

further yaml format issue fixed, missing brackets added

Qianqianye committed:
Update education resources

Qianqianye committed:
Add education resource

davepagurek committed:
Update some code blocks

limzykenneth committed:
Remove redundant reference entry

limzykenneth committed:
Remove Chinese translation of redundant entry

davepagurek committed:
Display code blocks in reference descriptions as blocks without highlighting

limzykenneth committed:
Remove duplicated access statement and fix links

shiffman committed:
fix link for accessibilty, add two resources

Qianqianye committed:
Add issue templates

Qianqianye committed:
Adjust issue template order

Qianqianye committed:
Update issue template style

Qianqianye committed:
Fix issue template

Qianqianye committed:
add p5.js Archived Website Japanese Translation

Qianqianye committed:
Add GitHub file links on Reference pages

github-actions[bot] committed:
Update p5.js to v1.10.0

Qianqianye committed:
Delete duplicated type entiries

DenisovichDev committed:
Fixed inline code issues

This fixes some porting issues with the page from the old website

DenisovichDev committed:
Fixed inline code issues in `normal()`

Qianqianye committed:
Build contributor docs with updated link path to one level up

Qianqianye committed:
Pull updates from p5.js repo contributor_docs folder

Qianqianye committed:
rename contributing_to_the_p5js_reference doc

Pull updates from p5.js repo contributor_docs folder

davepagurek committed:
Fix double-rendering of libraries page description

davepagurek committed:
Try to fix link path issues

davepagurek committed:
Handle more cases, update tests

davepagurek committed:
Update some more spots

limzykenneth committed:
Fix contributor docs script invalid import

davepagurek committed:
Fix tables and lists in reference items

Qianqianye committed:
Rebuild contribute pages from updated p5.js repo contributor docs

davepagurek committed:
Fix code blocks not wrapping

davepagurek committed:
Add experimentla icon + callout

davepagurek committed:
Update to handle dark mode

davepagurek committed:
Clone p5.js's latest release instead of main

davepagurek committed:
Update reference

sebastian-meier committed:
Adding parametric-design@fhp to educational ressources

github-actions[bot] committed:
Update p5.js to v1.11.0

Qianqianye committed:
Update utils.ts to v.1.11.0

Qianqianye committed:
Fix contributor doc after release

limzykenneth committed:
Fix types and function naming clash issue in reference

davepagurek committed:
Update reference for v11, fix locked version on p5 update

davepagurek committed:
Fix lingering references

davepagurek committed:
Update reference to show p5.sound

davepagurek committed:
Add route for sound reference

davepagurek committed:
Add new sound docs + get examples working

limzykenneth committed:
Update p5.sound reference

limzykenneth committed:
Fix reference build script not accounting for types reference

limzykenneth committed:
Fix tutorial reference links

shibomb committed:
fix: update old link to new website in tutorials

limzykenneth committed:
Delete duplicate files

zs-5 committed:
Fix typo (dpeth -> depth)

davepagurek committed:
Add script

zs-5 committed:
Fix typo (dpeth -> depth) in p5.Framebuffer.mdx

mohankumarhr committed:
changed the mispelling of paremeter to parameter

Souvik-Cyclic committed:
fix broken url

Please refer to issue #637

eltapir committed:
Update p5.5.yaml

eltapir committed:
Update p5.5.yaml

sarahciston committed:
update formatting T1&2

limzykenneth committed:
Update contributor docs

limzykenneth committed:
Update contributor list and reference

Stefterv committed:
added donate banner

Stefterv committed:
removed wallpaper

Stefterv committed:
hide banner

sarahciston committed:
update callout boxes, category config

SableRaf committed:
Update en.mdx to unhide old-site banner

The old-site banner was hidden to avoid having two banner on at the same time but we still want the old-site banner to be shown when the donation banner is closed so this is not the right approach.

wong-justin committed:
give class names a prefix on reference pages

fixes #650
before: blur reference page would have .blur CSS style applied
after: blur reference page has .reference_blur class name,
so there are no more collisions with CSS styles

hopefully this doesn't have any unintended side effects

calebfoss committed:
Remove keyboard trap (WCAG Success Criterion 2.1.2: No Keyboard Trap)

calebfoss committed:
Add exampleAuthors builder

calebfoss committed:
Work contributors into example layout, fix bugs in builder

calebfoss committed:
Fix bug

calebfoss committed:
Merge branch 'malay-vasa'

calebfoss committed:
Switch to relative URLs in example descriptions
